test/integration/puppetlabs/services/jruby/tasks_test.clj

```diff
@@ -140,8 +140,8 @@
         (testing "with the init task"
           (let [res (get-task-data "env1" "apache" "init")]
             (is (nil? (schema/check mc/TaskData res)))
-            (is (= "init.rb" (-> res :files first :name))
-            (is (re-matches #".*init\.rb" (-> res :files first :path))))))
```

Comment:
that's kinda terrifying

Reply:
I guess `(is)` returns true??

Reply:
Yeah not really sure what this was doing, but it started showing up as a syntax error only when run not in my IDE...

Reply:
The output of the second test was a parameter to the first `is` (it takes an optional string to output on failure)

Reply:
Oh I didn't know about the optional param. Joy.